### ISSUE TITLE
XrdHttp Configure phase is buggy if HTTPS is not desired

### ISSUE THREAD

**Issue by apeters1971** [xrootd/xrootd#2123]
```
if (httpsmode == hsmOff || (httpsmode == hsmAuto && !xrdctx && !httpsspec))
      {const char *why = (httpsmode == hsmOff ? "has been disabled!"
                         : "was not configured.");
       const char *what = Configed();

       eDest.Say("Config warning: HTTPS functionality ", why);
       httpsmode = hsmOff;

       if (what)
          {eDest.Say("Config failure: ", what, " HTTPS but it ", why);
           NoGo = 1;
          }
       return NoGo;
      }
```

If there is no certificate configuration and no desire for HTTPS, we return here with 0 and plug-ins are not loaded anymore. So the server runs without ext handlers!

**Comment by amadio** [comment]
Related to #2099.

**Comment by ccaffy** [comment]
Indeed, we are aware about that issue, I have to work on it ;)

**Comment by amadio** [comment]
Duplicate of #2099, fixed by #2253.
